docs/security.md

```diff
@@ -136,25 +136,33 @@ If you want to enable the authentication feature for KoP using the `OAUTHBEARER`
 
     For the `OAUTHBEARER` mechanism, you can use `AuthenticationProviderToken` or custom your authentication provider to process the access tokens from OAuth 2.0 server.
 
-    KoP provides a built-in `AuthenticateCallbackHandler` that uses the authentication provider of Pulsar for authentication. You need to configure the following properties in the `conf/kop-handler.properties` file.
+    KoP provides a built-in `AuthenticateCallbackHandler` that uses the authentication provider of Pulsar for authentication. You need to configure the following properties in the Pulsar broker's configuration file (`conf/broker.conf`)
```

Comment:
```suggestion
    KoP provides a built-in `AuthenticateCallbackHandler` that uses the authentication provider of Pulsar for authentication. You need to configure the following properties in the Pulsar broker's configuration file (e.g. `conf/broker.conf`)
```

It's better not to mention `conf/broker.conf` because we could also use `conf/standalone.conf` to configure standalone.